### ISSUE TITLE
ccex error

### ISSUE THREAD

**Issue by thijstriemstra** [kroitor/ccxt#77]
When you log into ccex with a fake api key and secret, and try to fetch the balance (which is obviously isn't there), it throws this traceback:
```
    balance = exchange.fetch_balance()
  File "/home/thijs/.virtualenvs/foo/lib/python3.5/site-packages/ccxt/__init__.py", line 5178, in fetch_balance
    for b in range (0, len (balances)):
TypeError: object of type 'NoneType' has no len()

```

Ccex returns the message below, so it should somehow check the APIKEY_INVALID value before checking the balance?

```
{"success":false,"message":"APIKEY_INVALID","result":null}
```

**Comment by kroitor** [comment]
Hi! Thanks! Will fix it very soon!

**Comment by kroitor** [comment]
Ok, the ccex will now throw an error in case of '!success', but there may be same symptoms with other exchanges, as we are still in the process of integrating proper unified error handling. The fix for ccex is available as of version 1.1.150.
